packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -402,8 +402,8 @@ async function getSingleOpBatch(
                 throw createGenericNetworkError(
                     "failedToRetrieveOpsFromStorage:TooManyRetries",
                     undefined,
-                    false /* canRetry */,
-                    undefined /* retryAfterSeconds */,
+                    { canRetry: false },
+                    (logger as any).driverVersion ?? "", // createOdspLogger stashes driverVersion on the logger
```

Comment:
I don't love this, but it was better to me than plumbing `driverVersion` through several functions with already large signatures.

Future improvements to consider here (low-pri, I think):
* Add an `IDriverTelemetryLogger` interface with `driverVersion` on it, and pass that around the driver
* Add a `props` field to `MonitoringContext`, and pass the mc around instead of the logger, and then read `props.driverVersion` here (wouldn't be as strongly typed, but better that `as any`)

Reply:
Tony had a good idea which is to expose `getTelemetryProperties` on the logger to expose the props it stamps on logs, so then I could get it properly from there instead of casting to `any`.  Will try that in a follow-up PR or open an issue if it's not as trivial as it sounds